nature-logger/Controllers/PlantsDetailedViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  PlantsDetailedViewController.swift
+//  nature-logger
+//
+//  Created by Frederik Helth on 15/12/2020.
+//  Copyright © 2020 Syddansk Universitet. All rights reserved.
+//
+
+import UIKit
+
+class PlantsDetailedViewController: UIViewController {
+
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var nameLabel: UILabel!
+    
+    var hero:Plants.Result?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        nameLabel.text = hero?.common_name
+        
+
+        var imageUrl = URL(string: "placeholder.png")!
```

Comment:
what if this placeholder won't get replaced? will it fail?

Reply:
No, this is the point of the placeholder, the default value is placeholder.png, if the image is not null it would get replaced